packages/cashier/src/Components/__tests__/cashier-container.spec.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { routes } from '@deriv/shared';
+import { fireEvent, render } from '@testing-library/react';
+import CashierContainer from '../cashier-container';
+
+const mockHistoryPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useHistory: () => ({
+        push: mockHistoryPush,
+    }),
+}));
+
+describe('<CashierContainer/>', () => {
+    const URL = `https://doughflow-test.4x.my/cashier/login.asp?Sportsbook=test&PIN=CR90000010&Lang=en&Password=20192eef9f1c8258354030632a913916fe331c8f&Secret=11eb12e081e93780306d7c8a7483748a&Action=DEPOSIT&udef1=EN&udef2=deriv&theme=light`;
```

Comment:
```suggestion
    const url = `https://doughflow-test.4x.my/cashier/login.asp?Sportsbook=test&PIN=CR90000010&Lang=en&Password=20192eef9f1c8258354030632a913916fe331c8f&Secret=11eb12e081e93780306d7c8a7483748a&Action=DEPOSIT&udef1=EN&udef2=deriv&theme=light`;
```

Reply:
@likhith-deriv Could you also hit an enter before writing `expects, so when we open the file can visually find the test expectations easier.

Reply:
Sure.. I will make these changes